resources/client/write-test.txt

```diff
@@ -28,3 +28,20 @@ Here comes the sun, and I say
 It's all right
 It's all right
 
+THIS FILE IS A SAMPLE FOR WRITING TO THE SERVER
```

Comment:
Appended text is not needed.

Reply:
Issue with the .gitignore not ignoring the resources/ folders. This is being worked on.